actionpack/lib/action_dispatch/system_testing/driver.rb

```diff
@@ -9,6 +9,7 @@ def initialize(name, **options, &capabilities)
         @screen_size = options[:screen_size]
         @options = options[:options]
         @capabilities = capabilities
+        @browser.preload
```

Comment:
This is picky but can you add a space before this so it's separate from setting the instance vars?

```suggestion

        @browser.preload
```